tests/test_magnetic_fields.py

```diff
@@ -1558,3 +1561,227 @@ def test_domm_field_is_nonzero_and_continuous_across_Z_0():
     Bnew = new_field.compute_magnetic_field(np.vstack([Rs, phis, Zs]).T)
 
     assert not np.any(np.isclose(Bnew[:, 1], 0))
+
+
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=10)
+@pytest.mark.unit
+def test_domm_W7AS():
+    """Test can get flux surfaces from W7-AS Dommashk coefficients."""
+    # W7-AS
+    # The coefficients are taken from IPP-Report IPP_0_48 by Dommaschk et al.
+    B0 = 1.00
+    # toroidal harmonics
+    ms = [
```

Comment:
if you want you can turn off black for these sections so it won't put everything on its own line: https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#ignoring-sections